docs/public/style.css

```diff
@@ -0,0 +1,5 @@
+.text-gradient em {
+  background: -webkit-linear-gradient(#eee, #333);
```

Comment:
Lol this is a ridiculous example to use I love it

Reply:
:mirror_ball: gotta funk it up